From b1669139f475ebe39ded6f7905f4c901f17eef83 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Wed, 24 Feb 2021 07:38:37 +0000
Subject: [PATCH] add missing static constexpr member definition

C++14 requires to definition of static constexpr members to
emit a linker symbol.
---
 .../angle/src/libANGLE/renderer/glslang_wrapper_utils.cpp      | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/third_party/angle/src/libANGLE/renderer/glslang_wrapper_utils.cpp b/third_party/angle/src/libANGLE/renderer/glslang_wrapper_utils.cpp
index 56b46e6..8f8158c 100644
--- third_party/angle/src/libANGLE/renderer/glslang_wrapper_utils.cpp
+++ third_party/angle/src/libANGLE/renderer/glslang_wrapper_utils.cpp
@@ -2069,6 +2069,9 @@ class SpirvTransformFeedbackCodeGenerator final : angle::NonCopyable
     spirv::IdRef mTransformFeedbackExtensionPositionId;
 };
 
+constexpr size_t SpirvTransformFeedbackCodeGenerator::kXfbDecorationCount;
+constexpr spv::Decoration SpirvTransformFeedbackCodeGenerator::kXfbDecorations[kXfbDecorationCount];
+
 void SpirvTransformFeedbackCodeGenerator::visitVariable(const ShaderInterfaceVariableInfo &info,
                                                         gl::ShaderType shaderType,
                                                         const spirv::LiteralString &name,
-- 
2.26.2

